### DISCUSSION TITLE
Unable to access Mattermost.

### DISCUSSION THREAD

**Discussion by Sai-Suraj-27** [helmholtz-analytics/heat#1027] [General]
First of all, I am extremely sorry to everyone for discussing this here, I will delete this discussion immediately when the problem is solved. For the last 2 days, I am not able to open my mattermost account. It is showing some error whenever I try to login 
![image](https://user-images.githubusercontent.com/87087741/189281107-ef21d903-150c-464f-afea-460fd5efa50a.png). 
I even mailed them explaining the problem. I don't know anywhere else to talk with my mentor @coquelin77. I want to tell you that I have submitted my final evaluation and this is the link to my final work product https://medium.com/@manosaisuraj12/gsoc22-work-product-submission-86deaf5058ba
I just want to make sure whether the medium article I wrote is good and ok for you as my final work product submission. As it is suggested by google to write a blog for the final submission I did that. I was afraid as I am not able to communicate with anyone through Mattermost so I am messaging here. Once again I am sorry for this. 
Thank you.

**Comment by ClaudiaComito** [comment]
Hi @SaiSuraj27 , thanks for bringing this up, @shahpratham has been running into this problem as well. Have you solved the problem @shahpratham ? 

@SaiSuraj27 I'll pass your message to Daniel so we're sure he sees it.

**Reply by shahpratham** [reply]
> Have you solved the problem @shahpratham ?

No, not with my browser, but still able to login via my phone.

**Reply by Sai-Suraj-27** [reply]
I received an email from them just an hour ago. I think they are fixing it. 
![image](https://user-images.githubusercontent.com/87087741/189334503-0fa76af9-3869-4031-ade5-f516781415ca.png)
@shahpratham if you are still facing the issue try to mail them.👍

**Comment by coquelin77** [comment]
Sorry for the issues with Mattermost @SaiSuraj27 the blog looks great!. The only small typo i found was in the acknowledgement of @ClaudiaComito and myself. It should read:

"Finally, I would once again like to thank my mentor [Daniel Coquelin](https://github.com/coquelin77), for helping me throughout the project duration, and also [Claudia Comito](https://github.com/ClaudiaComito) who helped me a lot during the initial application period."

Otherwise, great work!

**Reply by coquelin77** [reply]
also there is no reason to be sorry, this is a perfect place to bring up these issues :D

**Reply by Sai-Suraj-27** [reply]
Thank you for the immediate reply. I edited the article and made the suggested changes 👍.
